subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -2865,11 +2865,10 @@ void parse_mode_args_to_params(const struct shell *sh, int argc,
 		{"monitor", no_argument, 0, 'm'},
 		{"ap", no_argument, 0, 'a'},
 		{"softap", no_argument, 0, 'k'},
-		{"get", no_argument, 0, 'g'},
 		{"help", no_argument, 0, 'h'},
 		{0, 0, 0, 0}};
 
-	while ((opt = getopt_long(argc, argv, "i:smtpakgh",
```

Comment:
as @jerome-pouiller  pointed may be add `mode->oper = WIFI_MGMT_GET` as the explicit default?